.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "control-plane/PI"]
+	path = control-plane/PI
+	url = https://github.com/p4lang/PI.git
```

Comment:
Currently not public, so we should set things up to build without it.  Also, using an https: url rather than a git: one interacts poorly with public key use (particularly important for private repos.)